CONTRIBUTING.md

```diff
@@ -1,18 +1,18 @@
-# Welcome to Scaffold-ETH 2 Contributing Guide
+# Welcome to create-eth Contributing Guide
```

Comment:
what do you think about starting from uppercase letter? `Create-eth`

Reply:
At start I had `Create-eth`, changed it to match the command syntax trying to make it the "product name" too. 

But I'm happy changing it back if people prefer it, not attach to this silly idea at all haha

Reply:
I think I also like `create-eth`. But like 60/40 :D